ci-operator/step-registry/rosa/test/profile/rosa-test-profile-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: rosa-test-profile
+  steps:
+    pre:
+    - ref: rosa-cluster-setup
+    - ref: rosa-cluster-wait-ready-cluster
+    - ref: rosa-conf-idp-htpasswd
```

Comment:
There is an ENV control this right?

Reply:
This is just a demo workflow. We could remove/use it by our favorite.